### PR TITLE
Add delimiter option to lookupmap usage

### PR DESCRIPTION
**Pull request by chrode**

### CODE REVIEW

File: yo/app/scripts/directives/configuration.js [hunk 1]
```diff
@@ -373,12 +375,17 @@ angular.module('dmpApp')
          * Posts a file
          */
         $scope.uploadData = function() {
+
+            // Form scope gets somehow lost.
+            // This is a qnd to retrieve the relevant form value.
+            $scope.delimiter = $('#delimiter').val();
```
Comment (knutwalker) [inline yo/app/scripts/directives/configuration.js:381]: Ouch, qnd indeed. Do you have any idea, why the form is lost?

File: yo/app/scripts/services/util.js [hunk 2]
```diff
@@ -316,4 +316,31 @@ angular.module('dmpApp')
...
         };
+    })
+
+/**
+ * Provide utility to show alerts
+ */
+    .factory('showAlert', function($timeout) {
+
+        function show(scope, type, message, timeout) {
+            var alter = {
```
Comment (knutwalker) [inline yo/app/scripts/services/util.js:327]: yo, alder ;-)

File: yo/app/scripts/directives/configuration.js (hunk 1 above)
Comment (chrode) [inline yo/app/scripts/directives/configuration.js:381]: No idea. Tested around for quiet some time. Scope gets live updates while typing. But when submitting it seams to be using an old context.

File: yo/app/scripts/services/util.js (hunk 2 above)
Comment (chrode) [inline yo/app/scripts/services/util.js:327]: Was curious about that naming too. On second look that might be a typo ...

**Pull request merged**